### PR TITLE
docs: update dashboard development docs

### LINKED ISSUE #211: Update dashboard development documentation
It's currently still showing the old documentation

### PR DESCRIPTION
**Pull request by prydonius**
fixes #211

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -62,8 +37,7 @@ Now, to access the React app, simply run `kubeapps dashboard` as you usually
 would:
 
 ```
-cd ../kubeapps
-./kubeapps dashboard
+kubeapps dashboard --port=5000
```
Comment (arapulido) [inline dashboard/README.md]: why port 5000? wouldn't this be port 3000?

Comment (prydonius) [reply dashboard/README.md]: 3000 is the port the local node server runs on, 5000 is the port I've been using for the Dashboard proxy (whether I'm using telepresence or not to access it). No real reason, the random port is often a pain during development so I just picked one.

**Comment by prydonius** [comment]
ping @arapulido - mind taking another look?

**Comment by arapulido** [comment]
LGTM, feel free to merge. Thanks!

**Pull request merged**